src/outcomes.py

```diff
@@ -96,8 +116,10 @@ def borda_outcome(preferences: UnionArray) -> Result:
     return borda_points
 
 
-# Getting the outcomes for all the voting schemas
 def all_schemas_outcomes(s: UnionArray) -> Result:
```

Comment:
The return type should be a `dict` of `Results`?

Reply:
Yes my bad